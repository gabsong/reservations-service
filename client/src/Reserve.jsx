import React from 'react';

class Reserve extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <div>PriceHeader</div>
        <div>DatePicker</div>
        <div>GuestPicker</div>
        <div>PriceChart</div>
        <div>SubmitButton</div>
      </div>
    );
  }
}

export default Reserve;
