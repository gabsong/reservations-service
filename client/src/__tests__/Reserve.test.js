import React from 'react';
import renderer from 'react-test-renderer';
import Reserve from '../Reserve.jsx';

describe('<Reserve />', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Reserve />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
