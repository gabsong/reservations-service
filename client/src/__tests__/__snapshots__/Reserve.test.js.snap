// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reserve /> matches the snapshot 1`] = `
<div>
  <div>
    PriceHeader
  </div>
  <div>
    DatePicker
  </div>
  <div>
    GuestPicker
  </div>
  <div>
    PriceChart
  </div>
  <div>
    SubmitButton
  </div>
</div>
`;
